src/bosh_azure_cpi/spec/unit/utils/bosh_agent_util_spec.rb

```diff
@@ -33,77 +32,102 @@
     allow(config).to receive(:agent).and_return({'mbus' => 'http://u:p@somewhere'})
   end
 
-  context 'when using registry' do
-    let(:uses_registry) { true }
-    let(:expected_user_data) do
-      {
-          registry: { endpoint: registry_endpoint },
-          server: { name: instance_id },
-          dns: { nameserver: dns }
-      }
-    end
+  # TODO: coverage: need to implement the following specs to increase coverage
+  #   The uncovered code seems to be related to commit #c38b3026f9512491ffcb70be7088b4d3e319fdc0 (for issue #491).
+  describe '#encoded_user_data' do # rubocop:disable RSpec/RepeatedExampleGroupBody
+    it 'should return correct value'
+  end
 
-    it 'user data is for registry usage' do
-      user_data = agent_util.user_data_obj(
-        registry_endpoint,
-        instance_id,
-        dns,
-        agent_id,
-        network_spec,
-        environment,
-        vm_params,
-        config,
-      )
+  # TODO: coverage: need to implement the following specs to increase coverage
+  #   The uncovered code seems to be related to commit #c38b3026f9512491ffcb70be7088b4d3e319fdc0 (for issue #491).
+  describe '#encode_user_data' do # rubocop:disable RSpec/RepeatedExampleGroupBody
+    it 'should return correct value'
+  end
 
-      expect(user_data).to eq(expected_user_data)
-    end
+  # TODO: coverage: need to implement the following specs to increase coverage
+  #   The uncovered code seems to be related to commit #c38b3026f9512491ffcb70be7088b4d3e319fdc0 (for issue #491).
+  describe '#meta_data_obj' do # rubocop:disable RSpec/RepeatedExampleGroupBody
```

Comment:
Same: why is this test case empty?

Reply:
This code is related to #652 (areas of un-covered code which are already in master), and is technically unrelated to either #644 or this PR (other than the fact that they bring coverage down for this and all other PRs until #652 is fixed and coverage returns to 100%, which is stated as a requirement in the template for all PRs).

I had started an attempt to return coverage to 100% within this PR, but found that doing so would take too much time. So I left 9 Pending/unimplemented specs to be addressed later separately (see #652 for details).

Reply:
Any update on this PR? Would really like to take advantage of this feature in Azure.

Thanks!

Reply:
I will suggest to remove the empty test cases because we want to change the requirement to have 100% code coverage.

Reply:
removed as requested